theseus/utils/utils.py

```diff
@@ -112,6 +113,23 @@ def _compute(group_idx):
     return jacs
 
 
+# Returns a function that approximates a function's gradient with a central difference.
+def numeric_grad(
```

Comment:
Is `numeric_grad` a special case of the group version `numeric_jacobian` we currently have. We would have to pass th.Vector (torch.tensor) though or modify `numeric_jacobian` to also work with `np.ndarray`. Maybe not worth the hassle.

Reply:
Yeah, I think it's not worth the hassle, this one is much simpler to use when you have single input/output.